lib/ibm_watson/natural_language_understanding_v1.rb

```diff
@@ -96,22 +96,33 @@ def initialize(args = {})
 
     ##
     # @!method analyze(features:, text: nil, html: nil, url: nil, clean: nil, xpath: nil, fallback_to_raw: nil, return_analyzed_text: nil, language: nil, limit_text_characters: nil)
-    # Analyze text, HTML, or a public webpage.
-    # Analyzes text, HTML, or a public webpage with one or more text analysis features,
-    #   including categories, concepts, emotion, entities, keywords, metadata, relations,
-    #   semantic roles, and sentiment.
-    # @param features [Features] Specific features to analyze the document for.
+    # Analyze text.
+    # Analyzes text, HTML, or a public webpage for the following features:
+    #   - Categories
+    #   - Concepts
+    #   - Emotion
+    #   - Entities
+    #   - Keywords
+    #   - Metadata
+    #   - Relations
+    #   - Semantic roles
+    #   - Sentiment.
+    # @param features [Features] Analysis features and options.
     # @param text [String] The plain text to analyze. One of the `text`, `html`, or `url` parameters is
     #   required.
     # @param html [String] The HTML file to analyze. One of the `text`, `html`, or `url` parameters is
     #   required.
-    # @param url [String] The web page to analyze. One of the `text`, `html`, or `url` parameters is
+    # @param url [String] The webpage to analyze. One of the `text`, `html`, or `url` parameters is
     #   required.
-    # @param clean [Boolean] Remove website elements, such as links, ads, etc.
-    # @param xpath [String] An [XPath query](https://www.w3.org/TR/xpath/) to perform on `html` or `url`
-    #   input. Results of the query will be appended to the cleaned webpage text before it
-    #   is analyzed. To analyze only the results of the XPath query, set the `clean`
-    #   parameter to `false`.
+    # @param clean [Boolean] Set this to `false` to disable webpage cleaning. To learn more about webpage
+    #   cleaning, see the [Analyzing
+    #   webpages](/docs/services/natural-language-understanding/analyzing-webpages.html)
```

Comment:
Mamoon can you refactor these relative urls to use 
https://cloud.ibm.com/docs/services/...

Reply:
done